composer.json

```
@@ -23,14 +23,15 @@
     },
     "require": {
         "php": ">=7.1.3",
+        "ext-dom": "*",
         "jkphl/dom-factory": "^1",
         "jkphl/rdfa-lite-microdata": "^0.4.4",
-        "league/uri": "^5.0",
-        "mf2/mf2": "^0.4",
+        "league/uri": "^6.5",
```

Would it be possible to make this BC? Or what code change would be related to this change?

```suggestion
        "league/uri": "^5.0|^6.5",
```

---

Yeah that should be fine, doesn't appear that the used functions are modified